Description: Use format string argument with fprintf.
Author: Luca Falavigna <dktrkranz@debian.org>

--- LibVNCServer-0.9.9/client_examples/gtkvncviewer.c~	2012-05-04 16:19:00.000000000 +0200
+++ LibVNCServer-0.9.9/client_examples/gtkvncviewer.c	2013-11-03 09:40:26.573431061 +0100
@@ -563,7 +563,7 @@
 
 	time (&log_clock);
 	strftime (buf, 255, "%d/%m/%Y %X ", localtime (&log_clock));
-	fprintf (stdout, buf);
+	fprintf (stdout, "%s", buf);
 
 	vfprintf (stdout, format, args);
 	fflush (stdout);
