--- libvncserver-LibVNCServer-0.9.10/libvncclient/h264.c.orig	2014-12-06 20:18:09.065973169 +0100
+++ libvncserver-LibVNCServer-0.9.10/libvncclient/h264.c	2014-12-06 20:18:11.122639750 +0100
@@ -250,7 +250,7 @@
         va_sp_param_buf_id[i]  = VA_INVALID_ID;
         va_d_param_buf_id[i]   = VA_INVALID_ID;
     }
-    va_status = vaCreateSurfaces(va_dpy, width, height, VA_RT_FORMAT_YUV420, SURFACE_NUM, &va_surface_id[0]);
+    va_status = vaCreateSurfaces(va_dpy, VA_RT_FORMAT_YUV420, width, height, &va_surface_id[0], SURFACE_NUM, NULL, 0);
     CHECK_VASTATUS(va_status, "vaCreateSurfaces");
     for (i = 0; i < SURFACE_NUM; ++i) {
         DebugLog(("%s: va_surface_id[%d] = %p\n", __FUNCTION__, i, va_surface_id[i]));
