--- LibVNCServer-0.9.1/configure.ac.orig	2007-05-27 17:14:22.000000000 +0200
+++ LibVNCServer-0.9.1/configure.ac	2007-06-02 00:13:55.826402162 +0200
@@ -1,7 +1,7 @@
 # Process this file with autoconf to produce a configure script.
 AC_INIT(LibVNCServer, 0.9.1, http://sourceforge.net/projects/libvncserver)
 AM_INIT_AUTOMAKE(LibVNCServer, 0.9.1)
-AM_CONFIG_HEADER(rfbconfig.h)
+AC_CONFIG_HEADER(rfbconfig.h)
 AX_PREFIX_CONFIG_H([rfb/rfbconfig.h])
 
 # Checks for programs.
@@ -617,8 +617,8 @@
 AC_FUNC_STRFTIME
 AC_FUNC_VPRINTF
 AC_FUNC_FORK
-AC_CHECK_LIB(nsl,gethostbyname)
-AC_CHECK_LIB(socket,socket)
+AC_SEARCH_LIBS(gethostbyname,nsl)
+AC_SEARCH_LIBS(socket,socket)
 
 uname_s=`(uname -s) 2>/dev/null`
 if test "x$uname_s" = "xHP-UX"; then
@@ -637,7 +637,7 @@
 AM_CONDITIONAL(CYGIPC, test "$HAVE_CYGIPC" = "true")
 
 # Check if /dev/vcsa1 exists, if so, define LINUX
-AM_CONDITIONAL(LINUX, test -c /dev/vcsa1)
+AM_CONDITIONAL(LINUX, true)
 
 # Check for OS X specific header
 AC_CHECK_HEADER(ApplicationServices/ApplicationServices.h, HAVE_OSX="true")
@@ -675,10 +675,6 @@
 	test/Makefile
 	libvncserver-config
 	LibVNCServer.spec])
-# x11vnc only:
-if test "$build_x11vnc" = "yes"; then
-	AC_CONFIG_FILES([x11vnc/Makefile x11vnc/misc/Makefile])
-fi
 
 AC_CONFIG_COMMANDS([chmod-libvncserver-config],[chmod a+x libvncserver-config])
 AC_OUTPUT
