--- LibVNCServer-0.8.2/test/Makefile.am.orig	2006-04-26 17:39:25.000000000 +0200
+++ LibVNCServer-0.8.2/test/Makefile.am	2006-08-26 10:44:24.302838500 +0200
@@ -1,5 +1,5 @@
-AM_CFLAGS=-I $(top_srcdir) -g -Wall
-LDADD = ../libvncserver/libvncserver.a ../libvncclient/libvncclient.a @WSOCKLIB@
+AM_CFLAGS=-I $(top_srcdir) -Wall
+LDADD = ../libvncserver/libvncserver.la ../libvncclient/libvncclient.la @WSOCKLIB@
 
 if HAVE_LIBPTHREAD
 BACKGROUND_TEST=blooptest
--- LibVNCServer-0.8.2/contrib/Makefile.am.orig	2006-04-26 17:37:44.000000000 +0200
+++ LibVNCServer-0.8.2/contrib/Makefile.am	2006-08-26 10:42:43.588544250 +0200
@@ -1,5 +1,5 @@
 AM_CFLAGS = -I $(top_srcdir)
-LDADD = ../libvncserver/libvncserver.a @WSOCKLIB@
+LDADD = ../libvncserver/libvncserver.la @WSOCKLIB@
 
 noinst_PROGRAMS=zippy
 
--- LibVNCServer-0.8.2/libvncclient/Makefile.am.orig	2006-05-13 16:10:57.000000000 +0200
+++ LibVNCServer-0.8.2/libvncclient/Makefile.am	2006-08-26 10:55:07.563039750 +0200
@@ -1,15 +1,15 @@
-AM_CFLAGS=-g -I $(top_srcdir) -I. -Wall
+AM_CFLAGS=-I $(top_srcdir) -I. -Wall
 
-libvncclient_a_SOURCES=cursor.c listen.c rfbproto.c sockets.c vncviewer.c minilzo.c
+libvncclient_la_SOURCES=cursor.c listen.c rfbproto.c sockets.c vncviewer.c minilzo.c
 
 noinst_HEADERS=lzoconf.h minilzo.h
 
-rfbproto.o: rfbproto.c corre.c hextile.c rre.c tight.c zlib.c zrle.c ultra.c
+rfbproto.lo: rfbproto.c corre.c hextile.c rre.c tight.c zlib.c zrle.c ultra.c
 
 EXTRA_DIST=corre.c hextile.c rre.c tight.c zlib.c zrle.c ultra.c
 
-$(libvncclient_a_OBJECTS): ../rfb/rfbclient.h
+$(libvncclient_la_OBJECTS): ../rfb/rfbclient.h
 
-lib_LIBRARIES=libvncclient.a
+lib_LTLIBRARIES=libvncclient.la
 
 
--- LibVNCServer-0.8.2/libvncserver/Makefile.am.orig	2006-05-03 21:29:18.000000000 +0200
+++ LibVNCServer-0.8.2/libvncserver/Makefile.am	2006-08-26 10:44:04.749616500 +0200
@@ -1,4 +1,4 @@
-AM_CFLAGS=-g -Wall -I $(top_srcdir)
+AM_CFLAGS=-Wall -I $(top_srcdir)
 
 if WITH_TIGHTVNC_FILETRANSFER
 TIGHTVNCFILETRANSFERHDRS=tightvnc-filetransfer/filelistinfo.h \
@@ -39,9 +39,9 @@
 	draw.c selbox.c d3des.c vncauth.c cargs.c minilzo.c ultra.c scale.c \
 	$(ZLIBSRCS) $(JPEGSRCS) $(TIGHTVNCFILETRANSFERSRCS)
 
-libvncserver_a_SOURCES=$(LIB_SRCS)
+libvncserver_la_SOURCES=$(LIB_SRCS)
 
-lib_LIBRARIES=libvncserver.a
+lib_LTLIBRARIES=libvncserver.la
 
 if HAVE_RPM
 $(PACKAGE)-$(VERSION).tar.gz: dist
--- LibVNCServer-0.8.2/x11vnc/Makefile.am.orig	2006-07-09 03:48:31.000000000 +0200
+++ LibVNCServer-0.8.2/x11vnc/Makefile.am	2006-08-26 10:45:15.770055000 +0200
@@ -1,5 +1,5 @@
 AM_CFLAGS = -I $(top_srcdir)
-LDADD = ../libvncserver/libvncserver.a @WSOCKLIB@
+LDADD = ../libvncserver/libvncserver.la @WSOCKLIB@
 
 SUBDIRS = misc
 DIST_SUBDIRS = misc
--- LibVNCServer-0.7.1/vncterm/Makefile.am.orig	2004-12-02 01:07:45.000000000 +0100
+++ LibVNCServer-0.7.1/vncterm/Makefile.am	2005-10-23 09:52:53.550932000 +0200
@@ -4,7 +4,7 @@
 
 CFLAGS_ADD=-I..
 
-LDADD=../libvncserver/libvncserver.a @WSOCKLIB@
+LDADD=../libvncserver/libvncserver.la @WSOCKLIB@
 INCLUDES=-I.
 
 if LINUX
--- LibVNCServer-0.8.2/examples/Makefile.am.orig	2006-04-30 22:05:16.000000000 +0200
+++ LibVNCServer-0.8.2/examples/Makefile.am	2006-08-26 10:42:55.093263250 +0200
@@ -1,5 +1,5 @@
 AM_CFLAGS=-I $(top_srcdir) -g -Wall
-LDADD = ../libvncserver/libvncserver.a @WSOCKLIB@
+LDADD = ../libvncserver/libvncserver.la @WSOCKLIB@
 
 if OSX
 MAC=mac
--- LibVNCServer-0.8.2/client_examples/Makefile.am.orig	2006-04-26 17:37:44.000000000 +0200
+++ LibVNCServer-0.8.2/client_examples/Makefile.am	2006-08-26 10:42:27.551542000 +0200
@@ -1,5 +1,5 @@
 AM_CFLAGS=-I $(top_srcdir) -g -Wall
-LDADD = ../libvncclient/libvncclient.a @WSOCKLIB@
+LDADD = ../libvncclient/libvncclient.la @WSOCKLIB@
 
 if WITH_FFMPEG
 FFMPEG_HOME=@with_ffmpeg@
@@ -8,8 +8,8 @@
 MP3LAME_LIB=-lmp3lame
 endif
 
-vnc2mpg_CFLAGS=-I$(FFMPEG_HOME)/libavformat -I$(FFMPEG_HOME)/libavcodec -I$(FFMPEG_HOME)/libavutil
-vnc2mpg_LDADD=$(LDADD) $(FFMPEG_HOME)/libavformat/libavformat.a $(FFMPEG_HOME)/libavcodec/libavcodec.a $(MP3LAME_LIB) -lm
+vnc2mpg_CFLAGS=-I$(FFMPEG_HOME)/ffmpeg
+vnc2mpg_LDADD=$(LDADD) -lavformat -lavcodec $(MP3LAME_LIB) -lm
 
 FFMPEG_CLIENT=vnc2mpg
 endif
--- LibVNCServer-0.8.2/configure.ac.orig	2006-07-12 20:14:17.000000000 +0200
+++ LibVNCServer-0.8.2/configure.ac	2006-08-26 10:51:07.240020500 +0200
@@ -12,9 +12,7 @@
 	CCLD="$CC"
 fi
 AC_PROG_MAKE_SET
-AC_PROG_RANLIB
-AC_PATH_PROG([AR], [ar], [/usr/bin/ar],
-                  [$PATH:/usr/ccs/bin])
+AC_PROG_LIBTOOL
 
 # Options
 AH_TEMPLATE(WITH_TIGHTVNC_FILETRANSFER, [Disable TightVNCFileTransfer protocol])
@@ -455,8 +453,8 @@
 AC_FUNC_STRFTIME
 AC_FUNC_VPRINTF
 AC_FUNC_FORK
-AC_CHECK_LIB(nsl,gethostbyname)
-AC_CHECK_LIB(socket,socket)
+AC_SEARCH_LIBS(gethostbyname,nsl)
+AC_SEARCH_LIBS(socket,socket)
 
 uname_s=`(uname -s) 2>/dev/null`
 if test "x$uname_s" = "xHP-UX"; then
--- LibVNCServer-0.7.1/Makefile.am.orig	2005-03-07 17:33:56.000000000 +0100
+++ LibVNCServer-0.7.1/Makefile.am	2005-10-23 09:48:22.692108800 +0200
@@ -1,4 +1,4 @@
-CFLAGS=-g -Wall
+AM_CFLAGS=-Wall
 
 SUBDIRS=libvncserver examples contrib x11vnc vncterm classes libvncclient client_examples test
 DIST_SUBDIRS=libvncserver examples contrib x11vnc vncterm classes libvncclient client_examples test
